cmd/bosun/expr/funcs.go

```diff
@@ -732,6 +796,91 @@ func Band(e *State, T miniprofiler.Timer, query, duration, period string, num fl
 	return
 }
 
+func ShiftBand(e *State, T miniprofiler.Timer, query, duration, period string, num float64) (r *Results, err error) {
```

Comment:
By convention, in funcs.go, all the captial functions work with Sets, and the lowercase ones work within results within the set. So I am following that.